lib/node_modules/@stdlib/constants/float16/max-ln/README.md

```diff
@@ -0,0 +1,137 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2025 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# FLOAT16_MAX_LN
+
+> [Natural logarithm][natural-logarithm] of the maximum [half-precision floating-point number][ieee754].
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var FLOAT16_MAX_LN = require( '@stdlib/constants/float16/max-ln' );
+```
+
+#### FLOAT16_MAX_LN
+
+[Natural logarithm][natural-logarithm] of the maximum [half-precision floating-point number][ieee754].
+
+```javascript
+var bool = ( FLOAT16_MAX_LN === 11.09375 );
+// returns true
+```
+
+</section>
+
+<!-- /.usage -->
+
+<section class="examples">
+
+## Examples
+
+<!-- TODO: better example -->
+
+<!-- eslint no-undef: "error" -->
+
+```javascript
+var FLOAT16_MAX_LN = require( '@stdlib/constants/float16/max-ln' );
+
+console.log( FLOAT16_MAX_LN );
+// => 11.09375
+```
+
+</section>
+
+<!-- /.examples -->
+
+<!-- C interface documentation. -->
+
+* * *
+
+<section class="c">
+
+## C APIs
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/constants/float16/max_ln.h"
```

Comment:
We'll need to remove this macro as the value needs to be `stdlib_float16_t` which cannot be readily supported in a macro atm. You'll want to remove these docs, the header file, manifest.json, and update the package.json.